sections/semantics-forms.include

```diff
@@ -7348,7 +7348,7 @@ You cannot submit this form when the field is incorrect.</samp></pre>
     </dd>
   </dl>
 
-  The <{button}> element <a>represents</a> a button labeled by its contents.
+  The <{button}> element <a>represents</a> an interactive control, labeled by its contents, that allows for, if enabled,  user-triggered actions when activated.
 
```

Comment:
There are a lot of pieces to this sentence, which makes it messy. How about

"…a control allowing a user to trigger actions, when enabled. It is labeled by the content of the element."?

Also, what does the accessible name calculation say about labelling, and is that relevant here as an informative reference?